src/payment-flows/native/url.js

```diff
@@ -33,7 +33,16 @@ export function getNativePopupDomain({ props } : {| props : ButtonProps |}) : st
         return 'https://www.sandbox.paypal.com';
     }
 
-    return NATIVE_POPUP_DOMAIN[env];
+    const experimentOptions = {
+        sample: 1
```

Comment:
@mstuart @andregardi what do you think about having this at 0% to start? If we do 0% than it reduces risk while we verify it end to end in prod using the local storage opt-in:

```js
window.localStorage.setItem('enable_mobile_native_popup_domain', true);
```

Just thinking about migrating risk incase there's a problem with this logic.

```suggestion
        sample: 0
```

Reply:
I think that's a great idea. Especially because how sensitive this is :o

Reply:
We can quickly verify it's all working, then do a 1% release afterwards.

Reply:
Great suggestion @gregjopa! 
I just changed it to 0% now.